scene/2d/physics_body_2d.cpp

```diff
@@ -554,11 +554,12 @@ real_t RigidBody2D::get_inertia() const {
 
 void RigidBody2D::set_weight(real_t p_weight) {
 
-	set_mass(p_weight / 9.8);
+	set_mass(p_weight / real_t(GLOBAL_DEF("physics/2d/default_gravity", 98)) / 10);
```

Comment:
3.0 is compatibility breaking, so let's go ahead and make this cleaner, `physics/2d/default_gravity` should be `9.8` IMO, not `98`.